internal/pkg/template/workload.go

```diff
@@ -205,7 +205,6 @@ type LogConfigOpts struct {
 
 // HTTPTargetContainer represents the target group of a load balancer that points to a container.
 type HTTPTargetContainer struct {
-	Name string // Name of the container.
```

Comment:
Maybe remove `HTTPTargetContainer` and keep `HTTPTargetPort` only?

Reply:
I'll keep it as is for now as I've validated this template 😝